lib/registries/addon/drafts/draft/metadata/template.hbs

```diff
@@ -103,19 +106,24 @@
                     for={{subjectsFieldId}}
                     id='subjects'
                 >
-                    {{t 'registries.registration_metadata.subjects'}}
+                    <p local-class='DisplayText'>
+                        {{t 'registries.registration_metadata.subjects'}}
+                        <span local-class='Required'>*</span>
+                    </p>
                 </label>
                 <Subjects::Manager
                     @model={{this.draftManager.draftRegistration}}
                     @provider={{this.draftManager.draftRegistration.provider}}
                     @doesAutosave={{true}}
+                    @metadataChangeset={{this.draftManager.metadataChangeset}}
                     as |subjectsManager|
                 >
                     <Subjects::Widget
                         id={{subjectsFieldId}}
                         @subjectsManager={{subjectsManager}}
                     />
                 </Subjects::Manager>
+                <ValidationErrors @changeset={{this.draftManager.metadataChangeset}} @key={{'subjects'}} />
```

Comment:
Related to this. We have used inline loading indicator to denote `changeset` is validating. We may not need that here, but please consider adding passing `isValidating` flag to `ValidationErrors` here https://github.com/CenterForOpenScience/ember-osf-web/blob/60d3bf962c438a7536b41c09cf68777dfa24e1f3/lib/osf-components/addon/components/registries/review-metadata-renderer/label-display/template.hbs#L18-L22
A la https://github.com/CenterForOpenScience/ember-osf-web/blob/60d3bf962c438a7536b41c09cf68777dfa24e1f3/lib/osf-components/addon/components/registries/schema-block-renderer/read-only/files/template.hbs#L28-L32